docs/admin/logging/overview.md

```diff
@@ -130,6 +131,10 @@ By default, Docker uses the first 12 characters of the container ID to tag log m
 Refer to the [log tag option documentation](log_tags.md) for customizing
 the log tag format.
 
+`syslog-format` configures syslog protocol version to use when logging,
+Allowed value is rfc5424 in order to use RFC5424 syslog protocol. 
+Default is RFC3164 incompatible syslog protocol.
```

Comment:
I think both "rfc5424" and "rfc3164" should be accepted values here (and "rfc3164" the default value)